packages/fast-components-react-msft/src/call-to-action/call-to-action.stories.tsx

```diff
@@ -15,11 +15,21 @@ storiesOf("Call to action", module)
     .add("Justified", () => (
         <CallToAction appearance={CallToActionAppearance.justified}>Buy now</CallToAction>
     ))
+    .add("Justified Anchor", () => (
```

Comment:
Do the other lightweight anchor buttons have stories that represent this scenerio?